From a664acc78b160896f8bf3d44cc9db027f4bcb9dc Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Thu, 25 Nov 2010 15:07:55 +0800
Subject: [PATCH 194/505] ENGR00134055-1 Make update API requests to use new
 flag parameters

Update the flag parameters as the EDPC driver changed.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 test/pxp_lib_test/pxp_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/pxp_lib_test/pxp_test.c b/test/pxp_lib_test/pxp_test.c
index a287d1c..8638767 100644
--- a/test/pxp_lib_test/pxp_test.c
+++ b/test/pxp_lib_test/pxp_test.c
@@ -195,7 +195,7 @@ static void update_to_display(int left, int top, int width, int height, int wave
 	upd_data.update_region.top = top;
 	upd_data.update_region.height = height;
 	upd_data.temp = TEMP_USE_AMBIENT;
-	upd_data.use_alt_buffer = 0;
+	upd_data.flags = 0;
 
 	if (wait_for_complete) {
 		/* Get unique marker value */
-- 
1.8.0

